### ISSUE TITLE
BOLD brain mask cropping (multi echo data)

### ISSUE THREAD

**Issue by gb4944** [poldracklab/fmriprep#1442]
As shown in this image (grid pattern in top right corner is not a brain mask but a report issue), parts of the brain are missing from the mask:

![subjectbrainmask_me_fmriprep](https://user-images.githubusercontent.com/39733302/49882186-5bfd1c80-fe30-11e8-9e7c-f792ff0ff35c.PNG)

**Comment by effigies** [comment]
@emdupre How much of the original BOLD series would we need to reproduce the reference T2* map? Would, say, the first 50 volumes of each echo suffice?

**Comment by emdupre** [comment]
Sorry for the delay-- Yes, 50 volumes should be fine !

**Comment by effigies** [comment]
Thanks. @gb4944 Would you be willing to share the first 50 volumes of each of your echos? We can use them to improve the quality of our mask generation.

**Comment by gb4944** [comment]
@effigies Yes, is there a particular place where I could send or upload them to?

**Comment by effigies** [comment]
Any way you would like to share it is fine with me. If it's okay with you, we would publish those truncated series for our regression tests, which run with every commit to make sure that BOLD mask quality doesn't suffer as we make changes.

**Comment by gb4944** [comment]
@effigies 
The files are located here: https://drive.google.com/drive/folders/1oE2BW39TaYuj80iEh8SryUS58wwEYubK?usp=sharing
I included the full BOLD series in 'full', and the first 50 volumes only in 'truncated'. I hope this will be useful to you!

**Comment by oesteban** [comment]
@effigies @emdupre were you able to replicate this one?

**Comment by oesteban** [comment]
Hi @gb4944, I've tried to use your data, but in order to run fMRIPrep we need a T1w image for this subject too. Not sure you are still working on this problem.

**Comment by gb4944** [comment]
@oesteban Apologies, I've added the defaced T1w image here: [https://drive.google.com/open?id=1tDTzLRnf39YBf-AjPF4IpU6nNeIR0VkA](url)
